### ISSUE TITLE
400 Bad request when using item/field name

### ISSUE THREAD

**Issue by tomjohnburton** [1Password/ansible-onepasswordconnect-collection#58]
### Your environment

Collection Version: v2.2.0

Connect Server Version: 1.5.1

OS: MacOS

Ansible Version: ansible [core 2.13.0]

## What happened?

When I use the name of the secret instead of its uuid, I get a 400 error.

## What did you expect to happen?

Secret returned successfully

## Steps to reproduce
1. Set up a secret like this
```
- name: Find basic auth for monitoring cluster.
  item_info:
    item: cluster-basic-auth
    vault: monitoring
  no_log: true
  register: op_item
```

## Notes & Logs

```
Monday 27 June 2022  16:50:49 +0200 (0:00:00.362)       0:00:00.369 *********** 
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Server returned error with invalid JSON: HTTP Error 400: Bad Request"}```

**Comment by kpcraig** [comment]
Hey, tomjohnburton,

I've filed an issue in our internal tracking, thanks for bringing this to our attention!

**Comment by bachya** [comment]
@kpcraig Any news? This breaks the majority of my playbooks. I updated the API and sync engine to 1.5.6, but the same issue still occurs.

**Comment by kpcraig** [comment]
thanks for the ping - we're hoping to get some more person-hours onto these issues - i'll be sure to bring this one up specifically.

**Comment by ccauet** [comment]
Hey there, having the same issue, I wonder when you will release 2.2.1?

Side note: we use a lot of dynamic naming, e.g. the hostname is part of the item name, thus switching to UUID for selection is not possible without breaking a lot of stuff.

Thanks

**Comment by flokru** [comment]
Same here. I just installed the `main` branch locally but that is not an option for the production systems I manage.

**Comment by flokru** [comment]
But I can confirm that the fix solves the issue. Thanks for that!

**Comment by bachya** [comment]
@kpcraig Any word on releasing the fix for this?

**Comment by bachya** [comment]
@kpcraig Sorry for any hassle, but pinging again – can 2.2.1 be released? We're so close!

**Comment by volodymyrZotov** [comment]
Fixed in [2.2.1](https://github.com/1Password/ansible-onepasswordconnect-collection/releases/tag/v2.2.1) release.
